### PR TITLE
update user dashboard page ui and components

### PR DESCRIPTION
**Pull request by cloud9ight**
user main dashboard after login. 
ui and components done. 

hard code user info and showcase img and video right now, need connect to db asap

### CODE REVIEW

File: front-end/src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                .contact-link:hover {
+                    border-color: #FFD700;
+                }
+                .footer-link {
+                    color: #FFFFFF;
+                    text-decoration: none;
+                    transition: color 0.3s;
+                }
+                .footer-link:hover {
+                    color: #FFD700;
```
Comment (thogb) [inline front-end/src/components/Footer.jsx:71]: Use Mui components for theme consistency if can. No need to configure dark theme colours, dark theme by default.

File: front-end/src/app/(dashboard)/dashboard/page.tsx [hunk 2]
```diff
@@ -1,11 +1,248 @@
...
+                                </Typography>
+                            </Link>
+                            <Typography variant="body2" sx={{ marginTop: 1 }}>
+                                Access and modify your video library.
+                            </Typography>
+                        </Box>
+                    </Box>
+
+                   {/* Gallery Section */}
+<Typography variant="h6" sx={{ marginTop: 4 }}>
```
Comment (thogb) [inline front-end/src/app/(dashboard)/dashboard/page.tsx:192]: Need formatting. run "npm i" in root folder for later or install preitter. v

Comment (cloud9ight) [reply front-end/src/app/(dashboard)/dashboard/page.tsx:192]: will do later

File: front-end/src/components/Footer.jsx (hunk 1 above)
Comment (cloud9ight) [reply front-end/src/components/Footer.jsx]: done

File: front-end/src/app/(dashboard)/dashboard/page.tsx (hunk 2 above)
Comment (cloud9ight) [reply front-end/src/app/(dashboard)/dashboard/page.tsx]: done

**Pull request merged**